static/js/people.js

```diff
@@ -28,6 +28,10 @@ exports.init = function () {
 exports.init();
 
 exports.get_person_from_user_id = function (user_id) {
+    if (!people_by_user_id_dict.has(user_id)) {
+        blueslip.error('Unknown user_id for get_user_id: ' + user_id);
+        return undefined;
+    }
```

Comment:
@timabbott You made a comment on #3455, which was accidentally including this change, that you have cross-realm-user concerns here.  We might get a little blueslip spam here (including the tutorial), so I might want to make it a warning instead.